src/storage/secondary/rowset/disk_rowset.rs

```diff
@@ -120,8 +123,9 @@ impl DiskRowset {
         dvs: Vec<Arc<DeleteVector>>,
         seek_pos: ColumnSeekPosition,
         expr: Option<BoundExpr>,
+        end_sort_key: &[DataValue],
     ) -> StorageResult<RowSetIterator> {
-        RowSetIterator::new(self.clone(), column_refs, dvs, seek_pos, expr).await
+        RowSetIterator::new(self.clone(), column_refs, dvs, seek_pos, expr, end_sort_key).await
```

Comment:
Why we only pass `end_sort_key` here? I guess this function should have both `begin_sort_key` and `end_sort_key` as parameter?

Reply:
because i put the start row id in seek_pos, and will finish scanning when meet `end_sort_key`